contribs/gmf/src/d3-lidar-profile/lidarprofile.js

```diff
@@ -0,0 +1,37 @@
+goog.provide('gmf.lidarProfile');
+
+/**
+* Provides a D3js component to be used to draw an lidar point cloud
+* profile chart.
+* Requires access to a Pytree webservice: https://github.com/sitn/pytree
+*
+* @constructor
+* @param {Object} options from pytree
+*/
+
+gmf.lidarProfile = function(options) {
+
+  /**
+  * The instance of gmf.lidarProfile class used to access loader instance
+  * properties and methods from plot and measure instances
+  * @type {gmf.lidarProfile}
+  */
+  const parent = this;
```

Comment:
Can't you remove this constant and use directly `this` in constructor's call below ?

Reply:
Done

Reply:
Yes, fixed